apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md

```diff
@@ -56,7 +56,7 @@ SciPy is open source, and as such, this grant work will also be done in the open
 ## Transparent, open & sustainable infrastructure for conda-forge and bioconda
 **PIs: [Jaime Rodríguez-Guerra](https://github.com/jaimergp) (Quansight), [Wolf Vollprecht](https://github.com/wolfv) (Quantstack)**
```

Comment:
@jaimergp should this be `QuantStack` with capital `S` here as well?

Reply:
Yes QuantStack